src/store/getters.ts

```diff
@@ -7,11 +7,13 @@ import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 export default {
 	query( rootState: RootState ): QueryRepresentation {
 		return {
-			condition: {
-				propertyId: rootState.conditionRows[ 0 ].propertyData.id,
-				value: rootState.conditionRows[ 0 ].valueData.value,
-				propertyValueRelation: rootState.conditionRows[ 0 ].propertyValueRelationData.value,
-			},
+			conditions: [
```

Comment:
I think it would be ok to now implement a `.map()` here instead of just picking the first one

Reply:
Done

Reply:
Thanks!